common/src/main/java/gov/cms/ab2d/common/service/JobServiceImpl.java

```diff
@@ -130,7 +130,7 @@ public Resource getResourceForJob(String jobUuid, String fileName) throws Malfor
             throw new ResourceNotFoundException("No Job Output with the file name " + fileName + " exists in our records");
         }
 
-        Path file = Paths.get(fileDownloadPath + job.getJobUuid() + File.separator +  fileName);
+        Path file = Paths.get(fileDownloadPath, job.getJobUuid(), fileName);
```

Comment:
Was this the only place in the non-test code where we had this issue?

Reply:
That was the only one I could find. 
I did `search in path`  in the codebase for the string "efs.mount" and updated the ones that showed up